index.html

```diff
@@ -0,0 +1,359 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Exam project:Practical work html and css."
+    />
+    <title>Exam - Bhromaon</title>
+    <link rel="shortcut icon" href="favicon-32x32.png" type="image/png" />
+    <link rel="stylesheet" href="js/animation.js" />
+    <link rel="stylesheet" href="css/bootstrap-grid.min.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo&family=Josefin+Sans:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="menu_burger">
+        <div class="menu-btn">
+          <div class="line line__1"></div>
+          <div class="line line__2"></div>
+          <div class="line line__3"></div>
+        </div>
+
+        <div class="sub-menu-btn">
+          <div class="line line__1"></div>
+          <div class="line line__2"></div>
+        </div>
+        <img class="grogu" src="pngaaa.png" alt="TheChild" />
+
+        <ul class="nav-links">
+          <li class="link">
+            <a class="responsive_link" href="#">Who We Are</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">What We Do</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">Travel With Us</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h1>Bhromaon</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <p>Safe Travel Anywhere</p>
+          </div>
+        </div>
```

Comment:
Можно не заворачивать каждый блок в .row > .col-12, только если нужны колонки
```suggestion
        <h1>Bhromaon</h1>
        <p>Safe Travel Anywhere</p>
```